tools/test-api

```diff
@@ -77,6 +78,7 @@ with test_server_running(force=options.force, external_host='zulipdev.com:9981')
 
     test_the_api(client, nonadmin_client)
     test_generated_curl_examples_for_success(client)
+    test_api_description_for_success(client)
```

Comment:
I think this `test_api_description_for_success` test is unnecessary; `test-backend test_docs` already renders every API description.  So we can drop this commit.